### ISSUE TITLE
A non-numeric value encountered /forum/createtopic.php?method=newtopic PHP7

### ISSUE THREAD

**Issue by ivywe** [Geeklog-Plugins/forum#28]
/forum/createtopic.php?method=newtopic

error when using  PHP7 

```
An error has occurred:

This is being displayed as "Root Debugging" is enabled in your Geeklog configuration.

If this is a production website you must disable this option once you have resolved any issues you are investigating.

E_WARNING(2) - A non-numeric value encountered @ /path_to_public_html/forum/createtopic.php line 869
```

**Comment by eSilverStrike** [comment]
I haven't experienced this error with the forum.

What version of Geeklog?

Is this the forum 2.9.1 or the latest in the repository?

Is it when you visit the url or after you enter in some information and submit it?

**Comment by ivywe** [comment]
Yes,
https://github.com/Geeklog-Plugins/forum

> Is it when you visit the url or after you enter in some information and
submit it?
1. https://www.geeklog.jp/forum/viewtopic.php?showtopic=747
2. click new topic

thanks.

--
Ivy

2017-03-06 20:49 GMT+09:00 Tom <notifications@github.com>:

**Comment by eSilverStrike** [comment]
Hi Ivy,

I don't get the error. I do see that when I create a forum topic, the body of the topic is empty for some reason even though I added text. I am not sure why this is happening.

I tested this with your default theme and the denim theme. 

Is there any reason why you are not using the denim forum template files with the forum?

I have tested the forum on PHP 7.1.1 without issue on IIS

I am not sure if the issue is with your default theme, PHP 7.1.2, or the MariaDB server.
